### ISSUE TITLE
Remove dependency on bundler

### ISSUE THREAD

**Issue by Themitchell** [npm-ruby/npm-ruby#1]
When I install `npm-ruby` it appears to have a dependency on the `node-bundler` module. Is this really necessary?

**Comment by attilagyorffy** [comment]
That's right. We ship `node-bundler` built in to provide backwards compatibility for older applications that are in-transition state.
